internal/charmstore/store.go

```diff
@@ -802,7 +802,8 @@ var seriesScore = map[string]int{
 	"zesty":   8,
 	"artful":  9,
 	"cosmic":  10,
-	"disco":   0, // TODO: make this higher when disco is released
+	"disco":   11,
+	"eoan":    0, // TODO: make this higher when eoan is released
```

Comment:
Is it worth it? Why not just 12?

Reply:
12 would make it the default for any charm that was requested (and had a eoan charm available). That's probably not what you really want.